Reply/app/src/main/java/com/example/reply/ui/ReplyNavigation.kt

```diff
@@ -16,9 +16,57 @@
 
 package com.example.reply.ui
 
+import androidx.navigation.NavGraph.Companion.findStartDestination
+import androidx.navigation.NavHostController
+
 object ReplyDestinations {
     const val INBOX = "Inbox"
     const val ARTICLES = "Articles"
     const val DM = "DirectMessages"
     const val GROUPS = "Groups"
-}
\ No newline at end of file
+}
+
+class ReplyNavigationActions(navController: NavHostController) {
```

Comment:
I noticed JetNews used this actions implementation which seems very good for top level navigation. Hence I adopted it too. Let me know what you think on it @florina-muntenescu

Reply:
https://github.com/android/compose-samples/blob/main/JetNews/app/src/main/java/com/example/jetnews/ui/JetnewsNavigation.kt